bento/src/main/java/com/yelp/android/bento/utils/BentoAsyncInflationFlag.kt

```diff
@@ -0,0 +1,11 @@
+package com.yelp.android.bento.utils
+
+/**
+ * Global toggle for the async inflation feature in
+ * [com.yelp.android.bento.componentcontrollers.RecyclerViewComponentController]. When enabled,
+ * all RecyclerViewComponentControllers will try to inflate their component's views on a background
+ * thread. It's disabled by default.
+ */
+object BentoAsyncInflationFlag {
```

Comment:
If this a pattern we intend to keep, I would rather rename this class to BentoSettings, and rename `enabled` to `asyncInflationEnabled` so it can be used for other bento wide settings.